src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -2,77 +2,80 @@
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		int testValue = 4;
-		rhymer.countIn(testValue);
-
-		int result = rhymer.peekaboo();
-		Assert.assertEquals(testValue, result);
-	}
-
-	@Test
-	public void testCallCheck() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		boolean result = rhymer.callCheck();
-		Assert.assertEquals(true, result);
-
-		rhymer.countIn(888);
-
-		result = rhymer.callCheck();
-		Assert.assertEquals(false, result);
-	}
-
-	@Test
-	public void testIsFull() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		final int STACK_CAPACITY = 12;
-		for (int i = 0; i < STACK_CAPACITY; i++) {
-			boolean result = rhymer.isFull();
-			Assert.assertEquals(false, result);
-			rhymer.countIn(888);
-		}
-
-		boolean result = rhymer.isFull();
-		Assert.assertEquals(true, result);
-	}
-
-	@Test
-	public void testPeekaboo() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
-
-		int result = rhymer.peekaboo();
-		Assert.assertEquals(EMPTY_STACK_VALUE, result);
-
-		int testValue = 4;
-		rhymer.countIn(testValue);
-
-		result = rhymer.peekaboo();
-		Assert.assertEquals(testValue, result);
-		result = rhymer.peekaboo();
-		Assert.assertEquals(testValue, result);
-	}
-
-	@Test
-	public void testCountOut() {
-		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
-
-		int result = rhymer.countOut();
-		Assert.assertEquals(EMPTY_STACK_VALUE, result);
-
-		int testValue = 4;
-		rhymer.countIn(testValue);
-
-		result = rhymer.countOut();
-		Assert.assertEquals(testValue, result);
-		result = rhymer.countOut();
-		Assert.assertEquals(EMPTY_STACK_VALUE, result);
-	}
+    @Test
+    public void testCountIn() {
+        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+        int testValue = 4;
+        rhymer.countIn(testValue);
+
+        int result = rhymer.peekaboo();
+        Assert.assertEquals(testValue, result);
+    }
+
+    @Test
+    public void testCallCheck() {
+        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+        boolean result = rhymer.callCheck();
+        Assert.assertEquals(true, result);
+
+        rhymer.countIn(888);
+
+        result = rhymer.callCheck();
+        Assert.assertEquals(false, result);
+    }
+
+    @Test
+    public void testIsFull() {
+        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+        final int STACK_CAPACITY = 12;
+        for (int i = 0; i < STACK_CAPACITY; i++) {
+            boolean result = rhymer.isFull();
+            Assert.assertEquals(false, result);
+            rhymer.countIn(888);
+        }
+
+        boolean result = rhymer.isFull();
+        Assert.assertEquals(true, result);
+    }
+
+    @Test
+    public void testPeekaboo() {
+        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+        final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).